app/components/standup-deck.js

```diff
@@ -40,34 +41,39 @@ export default Component.extend(EKMixin, {
           if (this.startupIndex < this.startups.length - 1) {
             this.setTimerForState('startups');
             this.set('startupIndex', this.startupIndex + 1);
-            this.set(
-              'progress',
-              this.startupIndex / (this.startups.length - 1)
-            );
+            this.set('progress', this.startupIndex / totalLength);
           } else {
             this.set('startupIndex', 0);
             this.setTimerForState('incubators');
             this.set('state', 'incubators');
+            this.set('progress', this.startups.length / totalLength);
```

Comment:
`(this.startups.length - 1) + this.incubatorIndex` ? histoire de bien comprendre (et si ça marche)

Reply:
On vient de passer à incubators donc on est à `(this.startups.length - 1) + 1` d'où mon `this.startups.length`